bblfsh-dashboard/values.yaml

```diff
@@ -0,0 +1,29 @@
+# Default values for bblfsh-dashboard
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: bblfsh/dashboard
+  tag: v0.2.0
+  pullPolicy: IfNotPresent
+service:
+  name: bblfsh-dashboard
+  ## port where the dashboard will be listen at the LoadBalancer.
+  port: 80
+  ## loadBalancerIP if not set an ephermeral IP will be used.
+  loadBalancerIP:
+settings:
+  ## serverAddr is the address to a bblfsh server, if empty a bblfsh server
+  ## will be deployed by this chart. This server is deployed as a container
+  ## inside of the main pod, so if multiple replicas are configured multiple
+  ## bblfsh servers will be deployed
+  serverAddr:
+resources: {}
+
+# configuration for build-in server if, settings.serverAddr is empty.
+server:
```

Comment:
It may be a good idea to let the dashboard itself handle this dependency. Having the dashboard repo in charge of defining its server dependency configuration is more robust, and removes the need for deployers to know which server version matches with which dashboard version.

Reply:
How would it handle this dependency? I don't know how would the dashboard would let k8s know which version it should be deploying. There are not good ways of doing this dynamically in the template.

I can imagine that being handled externally by the deployment process whichever it is if there was a service or a parseable document that contains the matching versions of dashboard and bblfsh server

Reply:
The chart also allows connecting to an existing server, which could make sense in some deployment scenarios. In that case, there won't be any control of the versioning.

Reply:
@rporres I see what you mean, if it's not possible to extract dashboard dependencies from its repo, maybe this chart is the correct way to handle them, but maybe it should be part of the dashboard project, but I guess that's out of scope.